lib/checks/checks.go

```diff
@@ -641,35 +655,13 @@ func (r *checker) checkBandwidth(ctx context.Context, servers []Server) error {
 func (r *checker) collectTargets(ctx context.Context, server Server, requirements Requirements) ([]diskCheckTarget, error) {
 	var targets []diskCheckTarget
 
-	remote := &serverRemote{server, r.Remote}
-	// check if there's a system device specified
```

Comment:
Was too complicated for what it was doing - checking the server's state directory directly should suffice.